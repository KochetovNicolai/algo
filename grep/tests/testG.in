.*a(b|e)c.*|for
aaaa
abc
aec
oooooacoooo
oooaecooo
abcaecabc
forforfor