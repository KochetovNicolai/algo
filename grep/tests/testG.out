0	
0	
0	
0	

1	3 
0	
0	

1	3 
0	
0	

0	
0	
0	
0	
0	
0	
0	
0	
0	
0	
0	

4	6 7 8 9 
4	6 7 8 9 
4	6 7 8 9 
4	6 7 8 9 
0	
0	
0	
0	
0	

7	3 4 5 6 7 8 9 
4	6 7 8 9 
4	6 7 8 9 
4	6 7 8 9 
1	9 
1	9 
1	9 
0	
0	

7	3 4 5 6 7 8 9 
4	6 7 8 9 
4	6 7 8 9 
4	6 7 8 9 
1	9 
1	9 
1	9 
0	
0	

